Composer/packages/client/src/constants.tsx

```diff
@@ -527,9 +527,10 @@ export const defaultBotPort = 3979;
 export const defaultBotEndpoint = `http://localhost:${defaultBotPort}/api/messages`;
 
 const DAYS_IN_MS = 1000 * 60 * 60 * 24;
-
 export const SURVEY_PARAMETERS = {
-  daysUntilEligible: 5,
+  daysUntilEligible: 2,
   timeUntilNextSurvey: 90 * DAYS_IN_MS,
-  chanceToAppear: 0.15,
+  chanceToAppear: 0.3,
 };
+
+export const SURVEY_URL_BASE = 'https://microsoft.qualtrics.com/jfe/form/SV_bwlHGwEO2UDwo2F';
```

Comment:
Wondering if this should be an aka.ms link. Advantage would be that we could change the url without having to do a code change.

Reply:
That makes sense. I'll run it past the others and see what they think.